app/api/resources/user.py

```diff
@@ -255,3 +255,18 @@ def post(cls):
             'expiry': expiry.timestamp()
         }, 200
 
+
+# TODO: Add swagger docs
+@users_ns.route('home')
+@users_ns.response(200, home_response_body_model)
+class UserHomeStatistics(Resource):
+    @classmethod
+    @jwt_required
+    @users_ns.expect(auth_header_parser)
+    def get(cls):
+        """
+        :return: Statistics regarding the current user
+        """
+        user_id = get_jwt_identity()
+        stats = DAO.get_user_statistics(user_id)
+        return marshal(stats, home_response_body_model)
```

Comment:
Check what response is this API responding in case of success please, because this should return 200 as HTTP response. I tend to put aside with the marshal, but maybe its not necessary